dev/tools/gen_defaults/lib/text_field_template.dart

```diff
@@ -9,6 +9,13 @@ class TextFieldTemplate extends TokenTemplate {
 
   @override
   String generate() => '''
+TextStyle? _m3StateInputStyle(BuildContext context) => MaterialStateTextStyle.resolveWith((Set<MaterialState> states) {
```

Comment:
I'm totally unaware of this whole Material 3 token thing, is there somewhere I can read about it?

Reply:
This is the resource which explains it at a high level https://github.com/flutter/flutter/blob/master/dev/tools/gen_defaults/README.md . I mostly just looked into the other template files to get an idea of how to write this one. Not sure if there is a better resource.

Reply:
Thank you!